application/shoot.c

```diff
@@ -141,9 +142,12 @@ int16_t shoot_control_loop(void)
             }
             case SHOOT_AUTO_FIRE:
             {
-                shoot_control.trigger_speed_set = AUTO_FIRE_TRIGGER_SPEED;
                 break;
             }
+            case SHOOT_SEMI_AUTO_FIRE:
```

Comment:
Since SHOOT_AUTO_FIRE and SHOOT_SEMI_AUTO_FIRE all have nothing left to do, for clearance, we'd better combine them with the `default:` case